File: PASCAL/Cap04/c04ex07.pas
program c04ex07;

var
  Numero: Integer;

begin
  Write('Entre um numero inteiro: ');
  ReadLn(Numero);

  if (Numero >= 20) and (Numero <= 90) then
    WriteLn('O numero esta na faixa de 20 a 90.')
  else
    WriteLn('O numero esta fora da faixa de 20 a 90.');

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: GO/Cap04/c04ex07.go
// c04ex07.go

package main

import "fmt"

func main() {
  var Numero int

  fmt.Print("Entre um numero inteiro: ")
  fmt.Scan(&Numero)

  if (Numero >= 20 && Numero <= 90) {
    fmt.Println("O numero esta na faixa de 20 a 90.")
  } else {
    fmt.Println("O numero esta fora da faixa de 20 a 90.")
  }

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
